provider/oci/provider.go

```diff
@@ -206,34 +205,30 @@ func (e *EnvironProvider) Open(_ stdcontext.Context, params environs.OpenParams)
 	}
 
 	creds := params.Cloud.Credential.Attributes()
-	jujuConfig := common.JujuConfigProvider{
+	providerConfig := common.JujuConfigProvider{
 		Key:         []byte(creds["key"]),
 		Fingerprint: creds["fingerprint"],
 		Passphrase:  creds["pass-phrase"],
 		Tenancy:     creds["tenancy"],
 		User:        creds["user"],
-		OCIRegion:   creds["region"],
+		OCIRegion:   params.Cloud.Region,
 	}
```

Comment:
You still want to validate the providerConfig, we just didn't want to hide the validation inside of the `Config` method.